cfgov/regulations3k/models/pages.py

```diff
@@ -16,20 +18,103 @@
 from wagtail.wagtailcore.fields import StreamField
 from wagtail.wagtailcore.models import PageManager
 
-# Our RegDownTextField field doesn't generate a good widget yet
-# from regulations3k.models.fields import RegDownTextField
 from ask_cfpb.models.pages import SecondaryNavigationJSMixin
 from regulations3k.models import Part, Section
 from regulations3k.regdown import regdown
 from regulations3k.resolver import get_contents_resolver, get_url_resolver
+from regulations3k.scripts.integer_conversion import LETTER_CODES
```

Comment:
Long term it'd be nice to factor that into a utilities module in the regulations3k package instead of importing from scripts.

Reply:
√
Working toward that now.